sbws/util/stem.py

```diff
@@ -117,6 +122,33 @@ def _init_controller_socket(socket):
     return c
 
 
+def set_torrc_runtime_options(cont):
```

Comment:
Please call this function on line 74 as well, to remove duplicate code.

Reply:
yes, i forgot